channels/int.m3u

```diff
@@ -77,10 +79,16 @@ http://www.ast.tv/stream/2/master.m3u8
 https://live.creacast.com/cna/smil:cna.smil/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/tXTjWgP.png" group-title="Religious",Christian Youth Channel
 http://media.smc-host.com:1935/cycnow.com/cyc2/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="" group-title="XXX",Hustler TV
+http://catchup.iptv.iinet.rs/playlist.m3u8?id=177&token=dG9rZW49LVkzblctbGw0RUdzSWhVWWdDUlNiUjZMbGpBbkVUNzgmaWQ9NzImc2VjcmV0PW90dA==&DVR;?streamType=live
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="XXX",Hustler TV (Backup)
+http://188.35.9.26:2030/udp/047c
```

Comment:
Hustler is US channel: https://www.lyngsat.com/tvchannels/us/Hustler-HD-Europe.html

Reply:
Noted